docs/data/README.mdx

```diff
@@ -2,10 +2,28 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+The RPC, data indexers, analytical platforms, Galexie, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
```

Comment:
This list is pretty bloated and indigestible. I think we should rework to a generalized statement: 

> "Stellar supports tools that facilitate communication between users and applications on the Stellar network. Users can query...."

Reply:
Yeah I like that

Reply:
updated in a088c83120488cfcf7ef46830ab6621107828e2c